app/helpers/fhir_helper.rb

```diff
@@ -161,39 +176,54 @@ def patient_from_fhir(patient, default_jurisdiction_id)
   end
 
   # Build a FHIR US Core Race Extension given Sara Alert race booleans.
-  def to_us_core_race(white, black_or_african_american, american_indian_or_alaska_native, asian, native_hawaiian_or_other_pacific_islander)
+  def to_us_core_race(races)
     # Don't return an extension if all race categories are false or nil
-    return nil unless [white, black_or_african_american, american_indian_or_alaska_native, asian, native_hawaiian_or_other_pacific_islander].include?(true)
+    return nil unless races.values.include?(true)
 
     # Build out extension based on what race categories are true
     FHIR::Extension.new(url: 'http://hl7.org/fhir/us/core/StructureDefinition/us-core-race', extension: [
-      white ? FHIR::Extension.new(
+      races[:white] ? FHIR::Extension.new(
         url: 'ombCategory',
         valueCoding: FHIR::Coding.new(code: '2106-3', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'White')
       ) : nil,
-      black_or_african_american ? FHIR::Extension.new(
+      races[:black_or_african_american] ? FHIR::Extension.new(
         url: 'ombCategory',
         valueCoding: FHIR::Coding.new(code: '2054-5', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'Black or African American')
       ) : nil,
-      american_indian_or_alaska_native ? FHIR::Extension.new(
+      races[:american_indian_or_alaska_native] ? FHIR::Extension.new(
         url: 'ombCategory',
         valueCoding: FHIR::Coding.new(code: '1002-5', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'American Indian or Alaska Native')
       ) : nil,
-      asian ? FHIR::Extension.new(
+      races[:asian] ? FHIR::Extension.new(
         url: 'ombCategory',
         valueCoding: FHIR::Coding.new(code: '2028-9', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'Asian')
       ) : nil,
-      native_hawaiian_or_other_pacific_islander ? FHIR::Extension.new(
+      races[:native_hawaiian_or_other_pacific_islander] ? FHIR::Extension.new(
         url: 'ombCategory',
         valueCoding: FHIR::Coding.new(code: '2076-8', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'Native Hawaiian or Other Pacific Islander')
       ) : nil,
+      races[:race_other] ? FHIR::Extension.new(
+        url: 'ombCategory',
+        valueCoding: FHIR::Coding.new(code: 'OTH', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'Other')
+      ) : nil,
+      races[:race_unknown] ? FHIR::Extension.new(
+        url: 'ombCategory',
+        valueCoding: FHIR::Coding.new(code: 'UNK', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'Unknown')
+      ) : nil,
+      races[:race_refused_to_answer] ? FHIR::Extension.new(
+        url: 'ombCategory',
+        valueCoding: FHIR::Coding.new(code: 'ASKU', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'Refused to Answer')
+      ) : nil,
```

Comment:
Ok so actually I have some other thoughts on how this should be represented in FHIR. [Slack thread](https://mitrecorp.slack.com/archives/CCRMP31C7/p1613512721057800) for context.

Here are the issues as I see things, including with the Ethnicity :
- `"OTH"` is not in the [OMB Race Categories value set](https://www.hl7.org/fhir/us/core/ValueSet-omb-race-category.html#expansion), and so we can't use it like this.
- Neither NullFlavor code, `"UNK"` nor `"ASKU"` is in the [OMB Ethnicity Categories value set](https://www.hl7.org/fhir/us/core/ValueSet-omb-ethnicity-category.html#expansion), and so we can't use them like that for ethnicity.
- There is no ideal way to represent "Other" under the US Core Race Extension

So here are my suggestions:
- Rather than using `"UNK"` and `"ASKU"`, use [Data Absent Reason](https://www.hl7.org/fhir/extension-data-absent-reason.html) values `"unknown"` and `"asked-declined"`. [Here](https://mitrecorp.slack.com/archives/CCRMP31C7/p1613514934060600?thread_ts=1613512721.057800&cid=CCRMP31C7) is a helpful message from Reece about how to use Data Absent Reason. This suggestion applies to Race and Ethnicity.
- For "Other", just use an [`extension:text`](https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-ethnicity-definitions.html#Extension.extension:text) rather than `extension:ombCategory`, using `"other"` as the `valueString`. This suggestion only applies to Race.

Let me know your thoughts.

Reply:
addressed here: https://github.com/SaraAlert/SaraAlert/pull/689/commits/a64587cc4e781563136ee93f313d4d74d696d7fb#diff-402d451d1f7019322100522f6898c3153444edb8ddb963e181a457c84a2ca6b6

Reply:
> - Rather than using "UNK" and "ASKU", use Data Absent Reason values "unknown" and "asked-declined". Here is a helpful message from Reece about how to use Data Absent Reason. This suggestion applies to Race and Ethnicity.

I think this would be preferable to using `extension:text` for the `unknown` and `refused_to_answer`.

Reply:
updated: https://github.com/SaraAlert/SaraAlert/pull/689/commits/60eaa418e0bc7f7cb9ee8ca0decb8736ed1ef29d#diff-402d451d1f7019322100522f6898c3153444edb8ddb963e181a457c84a2ca6b6R210